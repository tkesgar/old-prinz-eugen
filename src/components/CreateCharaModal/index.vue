<template>
  <create-chara-modal
    @modal-hide="handleModalHide"
    @submit="handleSubmit"
  />
</template>

<script>
import CreateCharaModal from './view'

export default {
  components: {
    CreateCharaModal
  },
  methods: {
    handleModalHide (name) {
      return this.$bvModal.hide(`${name}_modal`)
    },
    handleSubmit (data) {
      console.log(data)
      console.log('uwu')
    }
  }
}
</script>
