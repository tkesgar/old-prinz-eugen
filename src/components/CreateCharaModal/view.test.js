import { mount } from '@vue/test-utils'
import localVue from '../../utils/test/local-vue'
import CreateCharaModal from './view'

describe('CreateCharaModal', () => {
  const wrapper = mount(CreateCharaModal, {localVue})

  test('should mount successfully', () => {
    expect(wrapper.isVueInstance()).toBe(true)
  })

  test('should match snapshot', () => {
    expect(wrapper.element).toMatchSnapshot()
  })
})
