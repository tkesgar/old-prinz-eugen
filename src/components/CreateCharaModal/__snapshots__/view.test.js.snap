// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CreateCharaModal should match snapshot 1`] = `<!---->`;
